### ISSUE TITLE
dom.clj gen-dom-macros function breaks compiled js (figwheel.main)

### ISSUE THREAD

**Issue by brianium** [fulcrologic/fulcro#427]
Line in question:

https://github.com/fulcrologic/fulcro/blob/develop/src/main/com/fulcrologic/fulcro/dom.clj#L192

Is the above line intentional? I am working on a fulcro project via figwheel.main, and the above line results in a `dom.js` file that contains something like the following:

```
20-09-19 20:40:45 DESKTOP-GBJJQFU INFO [com.fulcrologic.fulcro.dom:216] - Using Wrapped inputs
```

This is included literally, and results in an `Uncaught SyntaxError: Unexpected number` from JavaScript.

I am happy to fix this, just wasn't sure if it was intentionally left in and there is a preferred way to solve this other than removing the line.

**Comment by awkay** [comment]
Yeah, I'm not sure why stdout seems to get included in the code from macros. The line is an intentional notification, but yeah, we can take it out if it is ending up in js output, though I'd argue that should really be considered a bug in compiler.

**Comment by brianium** [comment]
If it helps, the compilation is following a somewhat unique path..

This is a fighweel.main build, passing a `:target :bundle` to the cljs compiler. The bundle is generated via webpack.

Perhaps the dust is still settling on certain aspects of `:target bundle`? I haven't looked into it though. 

Let me know if I can help in any way 🙇

**Comment by awkay** [comment]
I'm changing that to use stderr, perhaps that will help.

Try 3.3.5-SNAPSHOT please (or git SHA 8af237a3ae8007635a00d2c6a0eda0b65fb6257c) and let me know.

**Comment by brianium** [comment]
@awkay 

Just tried it out and the issue has been resolved 😄 

Thank you! 🙇

**Comment by awkay** [comment]
Released in 3.3.5
